Makefile

```diff
@@ -37,8 +37,6 @@ SSAS_ADMIN_CLIENT_SECRET := $(shell docker-compose run --rm ssas sh -c 'main --r
 #    export ACO_CMS_ID=A9999; make postman env=local
 # or
 #    ACO_CMS_ID=A9999 make postman env=local
-#
-# TODO: as part of BCDA-3692, generate credentials for E9994 and V994
```

Comment:
Remove this TODO without any changes because the caller can supply E9994/V994 in the MAKE command. The resulting CLIENT_ID and CLIENT_SECRET are used in other tests (BDT and Postman). But we're only validating a single ACO at time.

If we'd like to run Postman + BDT scan with the CEC/NGACO, we should update our travis.yml, but I do not think that it's necessary to repeat those tests. We're already verifying functionality in the smoke tests.

Reply:
Makes sense to me!